code/modules/mob/living/simple_mob/subtypes/animal/alien animals/teppi.dm

```diff
@@ -0,0 +1,1195 @@
+		else if(fireloss > 0)
+			adjustFireLoss(-0.5)
+		nutrition -= 5
+		heal_countdown = 5
+
+/mob/living/simple_mob/vore/alienanimals/teppi/proc/do_breeding(not_hungy)
+	if(!breedable || !not_hungy || prevent_breeding)
+		return
+	if(client)	//Player controlled teppi get a verb, so just do the countdown
+		if(baby_countdown > 0)
+			baby_countdown -= 1
+		return
+	if(baby_countdown > 0)
+		baby_countdown -= 1
+		return
+	else if(GLOB.teppi_count >= GLOB.max_teppi) //if we can't make more then we shouldn't look for partners, but we can be ready in case a slot opens
+		return
+	if(prob(1))
+		for(var/mob/living/simple_mob/vore/alienanimals/teppi/alltep in oview(1,src))
+			if(!teppi_adult || !alltep.teppi_adult || alltep.prevent_breeding) //Don't have babies if you or your partner is babies
+				continue
+			if(alltep.client) //Don't have babies if your partner is inhabited by a player.
+				continue
+			if(alltep)
+				new /mob/living/simple_mob/vore/alienanimals/teppi/baby(loc, src, alltep)
+				baby_countdown = 200
+				if(affinity[alltep.real_name])
+					return
+				handle_affinity(alltep, 30) //Mom and dad should like eachother when they do their business
+				alltep.handle_affinity(src, 30)
+				return
+/mob/living/simple_mob/vore/alienanimals/teppi/proc/teppi_sound()
+	if(!teppi_adult || client)
+		return
+	if(resting)
+		return
+	playsound(src, pick(teppi_sound), 75, 1)
+
+//Handles both growing up from a baby and also passing parent details to new babies.
+/mob/living/simple_mob/vore/alienanimals/teppi/New(newloc, teppi1, teppi2)
+	GLOB.teppi_count += 1
+	if(teppi1 && !teppi2)
+		inherit_from_baby(teppi1)
+	else if (teppi1 && teppi2)
+		inherit_from_parents(teppi1, teppi2)
+	..()
+
+/mob/living/simple_mob/vore/alienanimals/teppi/Destroy()
+	GLOB.teppi_count -= 1
+	friend_zone = null
+	active_ghost_pods -= src
+	ai_holder.leader = null
+	return ..()
+
+/mob/living/simple_mob/vore/alienanimals/teppi/Login()
+	..()
+	to_chat(src, "<span class='notice'>While you may have access to galactic common, this is purely meant for making it so you can understand people in an OOC manner, for facilitating roleplay. You almost certainly should not be speaking to people or roleplaying as though you understand everything everyone says perfectly, but it's not unreasonable to be able to intuit intent and such through people's tones when they speak. Teppi are kind of smart, but they are animals, and should be roleplayed as such.</span> <span class='warning'>ADDITIONALLY, you have the ability to produce offspring if you're well fed enough every once in a while, and the ability to disable this from happening to you. These verbs exist for to preserve the mechanical functionality of the mob you are playing. You should be aware of your surroundings when you use this verb, and NEVER use it to prefbreak or be disruptive. If in doubt, don't use it.</span> <span class='notice'>Also, to note, AI Teppi will never initiate breeding with player Teppi.</span>")
```

Comment:
Pls change: No need to do this, there is already a var on simple_mobs called `player_msg` that is printed to players when they log in. It's null by default and prints nothing. Just put your message in there on teppi.

Reply:
For this one, I did this in two steps because player_msg doesn't seem to allow any formatting. I put ICish info in the player_msg, and OOCish info here, because it can be formatted.

Reply:
It should allow formatting, it's just in a to_chat like this is. Test again? Make sure to include the spans and stuff from your current one.

Reply:
I fiddled with it and got it working!